pub-kube-config-files/upp-aggregate-healthcheck.yaml

```diff
@@ -0,0 +1,126 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: upp-aggregate-healthcheck
+  labels:
+    app: upp-aggregate-healthcheck
+    visualize: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: upp-aggregate-healthcheck
+  template:
+    metadata:
+      labels:
+        app: upp-aggregate-healthcheck
+        version: "latest"
+        visualize: "true"
+    spec:
+      containers:
+      - name: aggregate-healthcheck
+        image: coco/upp-aggregate-healthcheck:v1.0.0-rc31
```

Comment:
I know that my version ok agg-hc yaml is not up-to-date with the one on the github, but I will leave you some comments here to make this ok:
- please use the v1.1.1-rc4 version of this image since this includes the improved agg-hc and graphite integration